README.md

```diff
@@ -52,6 +52,12 @@ cd experiment
 bin/physlr-make physical-map lr=linkedreads ref=reference protocol=stlfr
 ```
 
+If you provide a reference genome, Physlr automatically maps the physical map to the input reference and outputs a PAF file. You can run the following command which uses the PAF file to calculate assembly-like quality metrics for the physical map.
```

Comment:
I'd rearrange these sentences to make it more clear that providing a reference allows physlr to calculate the quality metrics for physical map. 
Something like:

If you provide a reference genome (with `ref`), physlr automatically calculates assembly-like quality metrics for the physical map. Physlr automatically maps the physical map to the input reference, and outputs a PAF file that is parsed to calculate these metrics